keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -83,12 +83,23 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
     `lowercase=False`, `strip_accents=False` and `split=False`. In
     this case, inputs should be pre-split string tensors or ragged tensors.
 
+    Tokenizer outputs can either be padded and truncated with a
+    `sequence_length` argument, or left un-truncated. The exact output will
```

Comment:
Not a blocking comment - this part appears to be duplicated in all tokenizer subclasses, do we want to move it into the base tokenizer?

Reply:
I'm worried about discover-ability. If we only document this on the base class, no one will find it. We this to show up in the documentation for every tokenizer.

Also not a contract, it's just a convention.

Reply:
We can let this PR go for now, but we can possibly do similar treatment like: https://github.com/keras-team/keras/blob/master/keras/optimizers/optimizer_experimental/adadelta.py#L140

we can do it as a followup

Reply:
Ah yeah makes sense. Let's consider that as a follow up.

I don't think it is fair to demand all tokenizers support `sequence_length`, in fact, the example we give on the base class does not. It's more a convention.

Reply:
Though maybe we could consider a redesign here where that logic was contained on the base class. Will keep thinking on that. Anyway a follow up :)